### PR TITLE
Update library import checks

### PR DESCRIPTION
**Pull request by eirikurj**
## Purpose
Currently, the library import check suppresses useful information why the library could not be imported. This PR updates the import check to print out the full error message. 

## Expected time until merged
No rush.

## Type of change

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (non-backwards-compatible fix or feature)
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no API changes)
- [ ] Documentation update
- [ ] Maintenance update
- [ ] Other (please describe)

## Testing

Compile the module with and without the fix, with a broken library (e.g., by commenting out `CGNS_LINKER_FLAGS` in the config).

## Checklist

- [x] I have run `flake8` and `black` to make sure the Python code adheres to PEP-8 and is consistently formatted
- [ ] I have formatted the Fortran code with `fprettify` or C/C++ code with `clang-format` as applicable
- [ ] I have run unit and regression tests which pass locally with my changes
- [ ] I have added new tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation

### CODE REVIEW

**Review by nwu63 — APPROVED** [review]
LGTM

**Review by sseraj — COMMENTED** [review]
The complex build is failing. It looks like a path issue with the import test.

**Comment by eirikurj** [comment]
Ugh, right, the `PATH` is probably missing the directory where the lib is. Will fix.

File: src/f2py/importTest.py [hunk 1]
```diff
@@ -1,14 +1,29 @@
...
 import sys
 
-name = "libidwarp"
-print("Testing if module %s can be imported..." % name)
-import_cmd = "import %s" % name
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    "name",
+    type=str,
+    help="Library name (example: libpackage.so). Note: This script must be run in the same dir as the library.",
```
Comment (sseraj) [inline src/f2py/importTest.py:10]: The note is not true anymore, right?

Comment (sseraj) [reply src/f2py/importTest.py:10]: Actually, I think I misinterpreted this. It makes sense now.

**Review by sseraj — APPROVED** [review]

**Review by nwu63 — APPROVED** [review]

**Review by sseraj — APPROVED** [review]

**Pull request merged**